CRM/Core/Payment/Stripe.php

```diff
@@ -669,4 +669,23 @@ function doRecurPayment(&$params, $amount, $stripe_customer) {
   function doTransferCheckout(&$params, $component) {
     CRM_Core_Error::fatal(ts('Use direct billing instead of Transfer method.'));
   }
+
+  /**
+   * Stripe payment instrument validation.
+   *
+   * @param array $values
+   * @param array $errors
+   */
+  public function validatePaymentInstrument($values, &$errors) {
+    $mandatoryFields = $this->getMandatoryFields();
+
+    // set credit_card_number and cvv2 as not required as we have stripe_token
```

Comment:
Let me suggest a potentially better comment:
`// We do not require neither credit_card_number nor cvv2 because we use stripe_token`